transforms/language/html2parquet/kfp_ray/pipeline_definitions.yaml

```diff
@@ -1,7 +1,7 @@
 pipeline_parameters:
     name: "html2parquet"
     description: "Pipeline for html2parquet task"
-    script_name: "html2parquet_transform_ray.py"
+    script_name: "-m dpk_html2parquet.ray.transform"
```

Comment:
`script` to `module`

Reply:
@daw3rd this will unecessary break the downstream code. This is still a python script that is invoked differently. Anyway, I will leave it to @eres / @roytman  to decide.

Reply:
In this case, let's keep it as script_name:. The fewer changes we make for this first iteration, the fast we will be able to move, test and iterate. :-)